threshsign/src/ThresholdSignaturesTypes.cpp

```diff
@@ -42,11 +43,13 @@ Cryptosystem::Cryptosystem(const std::string& sysType,
 
 // Helper function to generateNewPseudorandomKeys.
 IThresholdFactory* Cryptosystem::createThresholdFactory() {
-  if (type_ == MULTISIG_BLS_SCHEME || forceMultisig_) {
+  if (type_ == MULTISIG_BLS_SCHEME && forceMultisig_) {
```

Comment:
at the time it was intentionally ||

Reply:
I know, but "multisig" is no longer unique to bls.
Should this still be reverted?